.github/workflows/CD.yml

```diff
@@ -407,8 +423,12 @@ jobs:
       - name: Run testing suite on ecs
         if: steps.release-validation-ecs.outputs.cache-hit != 'true'
         run: |
-          if [[ -f testing-framework/terraform/testcases/${{ matrix.testcase }}/parameters.tfvars ]] ; then opts="-var-file=../testcases/${{ matrix.testcase }}/parameters.tfvars" ; else opts="" ; fi
-          cd testing-framework/terraform/ecs && terraform init && terraform apply -auto-approve -lock=false $opts -var="ecs_launch_type=${{ matrix.launch_type }}" -var="aoc_version=${{ github.event.inputs.version }}" -var="aoc_image_repo=$IMAGE_LINK" -var="testcase=../testcases/${{ matrix.testcase }}"
+          opts=""
+          if [[ -f testing-framework/terraform/testcases/${{ matrix.testcase }}/parameters.tfvars ]] ; then
+            opts="-var-file=../testcases/${{ matrix.testcase }}/parameters.tfvars"
+          fi
+          cd testing-framework/terraform/ecs && terraform init \
+            && terraform apply -auto-approve -lock=false $opts -var="region=$INTEGRATION_TEST_SECOND_REGION" -var="ecs_launch_type=${{ matrix.launch_type }}" -var="aoc_version=${{ github.event.inputs.version }}" -var="aoc_image_repo=$IMAGE_LINK" -var="testcase=../testcases/${{ matrix.testcase }}" -var="cortex_instance_endpoint=$INTEGRATION_TEST_AMP_SECOND_REGION"
```

Comment:
You could probably remove the \
            &&
since it is using run: |

Reply:
Reasonable. Fixed it in the newest commit though.

Reply:
This is done everywhere.